### PR TITLE
[CI] Azure pipeline migration. support nightly build on all platforms & branch master

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
for sonic image nightly build
#### How I did it
pipeline: https://dev.azure.com/mssonic/build/_build?definitionId=72&_a=summary
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: .azure-pipelines/official-build.yml [hunk 1]
```diff
@@ -15,8 +15,18 @@ trigger: none
 pr: none
 
 stages:
-- stage: Build
+- stage: Build_barefoot
```
Comment (lguohan) [inline .azure-pipelines/official-build.yml:18]: why are we having so many stages?

Comment (liushilongbuaa) [reply .azure-pipelines/official-build.yml:18]: Just for beter result showing. Here is an example: https://dev.azure.com/mssonic/build/_build?definitionId=72

Comment (liushilongbuaa) [reply .azure-pipelines/official-build.yml:18]: For [nephos](https://sonic-jenkins.westus2.cloudapp.azure.com/job/nephos/job/buildimage-nephos-all/), the build for master branch breaks for a month. If we keep them in one stage, the result is always fail.

Comment (lguohan) [reply .azure-pipelines/official-build.yml:18]: stage is designed to have dependency. if we want to make the build independent, then we can create multiple pipeline and each pipeline handle one platform.

Comment (liushilongbuaa) [reply .azure-pipelines/official-build.yml:18]: we can remove dependency by:   dependsOn: []

**Pull request closed without merging**